packages/packagers/css/src/CSSPackager.js

```diff
@@ -99,3 +112,63 @@ export default (new Packager({
     });
   },
 }): Packager);
+
+async function processCSSModule(
+  options,
+  bundleGraph,
+  bundle,
+  asset,
+  media,
+): Promise<[Asset, string, ?Buffer]> {
+  let ast: Root = postcss.fromJSON(nullthrows((await asset.getAST())?.program));
+
+  let usedSymbols = bundleGraph.getUsedSymbols(asset);
+  let localSymbols = new Set(
+    [...asset.symbols].map(([, {local}]) => `.${local}`),
+  );
+  let usedLocalSymbols =
+    // we have to still support the more common default imports
```

Comment:
Wonder if we should add a warning here so people know to switch from default imports to namespaces?

Reply:
A warning would be nice, but at this point, there is no way to differentiate a default import from this case:
```js
import * as styles from "..."

console.log(styles.default);
```

Reply:
Can we add some kind of metadata to the dependency to mark this? I think a lot of projects might not see any benefit from this because they use default imports...

Reply:
<img width="1090" alt="Bildschirmfoto 2020-12-30 um 13 09 47" src="https://user-images.githubusercontent.com/4586894/103350443-4d7a1000-4aa0-11eb-9ee6-6a4f07f6f008.png">